pkg/operator/observe_config.go

```diff
@@ -292,6 +294,50 @@ func ConvertJSON(o interface{}) (interface{}, error) {
 	return ret, nil
 }
 
+// observeBuildAdditionalCA observes if the additional trusted CA is available.
+func observeBuildAdditionalCA(listers Listers, observedConfig map[string]interface{}, operatorConfig *operatorconfigv1alpha1.OpenShiftControllerManagerOperatorConfig) (map[string]interface{}, error) {
+	build, err := listers.buildConfigLister.Get("cluster")
+	if errors.IsNotFound(err) {
+		operatorConfig.Spec.AdditionalTrustedCA = nil
+		return observedConfig, nil
+	}
+	if err != nil {
+		return nil, err
+	}
+	if len(build.Spec.AdditionalTrustedCA.Name) == 0 {
+		operatorConfig.Spec.AdditionalTrustedCA = nil
+		return observedConfig, nil
+	}
+	cm, err := listers.clusterConfigConfigMapLister.ConfigMaps(openshiftConfigNamespaceName).Get(build.Spec.AdditionalTrustedCA.Name)
+	if errors.IsNotFound(err) {
+		operatorConfig.Spec.AdditionalTrustedCA = nil
+		glog.Warningf("referenced configMap %s/%s for additional trusted certificate authorities does not exist",
+			openshiftConfigNamespaceName,
+			build.Spec.AdditionalTrustedCA.Name)
+		return observedConfig, nil
```

Comment:
If build CRD references a ConfigMap that doesn't exist, log the warning and treat it as if no CA had been defined.